rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf

```diff
@@ -177,6 +177,23 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/post.php" \
 		ctl:ruleRemoveTargetByTag=attack-sqli;ARGS:post_title,\
 		ctl:ruleRemoveTargetByTag=CRS;ARGS:content"
 
+#
+# [ Helpers ]
+#
+
+# /wp-admin/load-scripts.php?c=0&load%5B%5D=hoverIntent,common,
+# admin-bar,wp-ajax-response,jquery-color,wp-lists,quicktags,
+# jquery-query,admin-comments,svg-painter,heartbeat,&load%5B%5D=
+# wp-auth-check,wp-a11y,wplink,jquery-ui-core,jquery-ui-widget,
+# jquery-ui-position,jquery-ui-menu,jquery-ui-autocomplete&ver=4.6.1
+SecRule REQUEST_FILENAME "@endsWith /wp-admin/load-scripts.php" \
+	"id:9002800,\
+	phase:2,\
+	t:none,\
+	nolog,\
+	pass,\
+	ctl:ruleRemoveTargetById=942430;ARGS:load[]"
```

Comment:
so this is loading as an array, is anything every pushed into the array, if so it may be annoying. also this seems pretty dangerous.